test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -135,12 +158,9 @@ final class Util implements AWTEventListener {
     public static final int CODE_FIRST_SUCCESS = 0x2;
     public static final int CODE_SECOND_SUCCESS = 0x2;
     public static final int CODE_FAILURE = 0x1;
-
-    public static final int FRAME_ACTIVATION_TIMEOUT = 3000;
+    public static final int FRAME_ACTIVATION_TIMEOUT = 1000;
 
     static final Object SYNC_LOCK = new Object();
-    static final int MOUSE_RELEASE_TIMEOUT = 1000;
-
     static final Util theInstance = new Util();
```

Comment:
Could you please add the blank lines as they were so that this piece of code looks like this:

```java
    public static final int CODE_FIRST_SUCCESS = 0x2;
    public static final int CODE_SECOND_SUCCESS = 0x2;
    public static final int CODE_FAILURE = 0x1;

    public static final int FRAME_ACTIVATION_TIMEOUT = 1000;

    static final Object SYNC_LOCK = new Object();

    static final Util theInstance = new Util();
```

With blank lines separating logical blocks of fields.